app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -461,6 +470,13 @@ class LuciBuildService {
     log.info('Published a request with ${buildRequests.length} builds');
   }
 
+  /// Waits for GoB mirror to get the commit before publishing build requests.
+  Future<bool> isGobMirrorAtCommit({
+    required Commit commit,
+  }) async {
+    return true;
```

Comment:
check will be done here. Does the Commit object have everything necessary to construct the GoB URL?